plugins/woocommerce/tests/e2e-pw/tests/admin-analytics/analytics-overview.spec.js

```diff
@@ -72,19 +72,23 @@ test.describe( 'Analytics pages', () => {
 			await page.goto(
 				'wp-admin/admin.php?page=wc-admin&path=%2Fanalytics%2Foverview'
 			);
-			// check the top section (Performance)
-			await page.click(
-				'//button[@title="Choose which analytics to display and the section name"]'
-			);
+			// check the top section
+			await page
+				.locator( 'button.woocommerce-ellipsis-menu__toggle' )
+				.first()
+				.click();
 			await expect( page.locator( 'text=Move up' ) ).not.toBeVisible();
 			await expect( page.locator( 'text=Move down' ) ).toBeVisible();
+			await page.keyboard.press( 'Escape' );
```

Comment:
In the daily smoke test site, the menu is too long such that it blocks the menu buttons of sections below it. Pressing ESC will make sure that the menu is closed before attempting to click on another ellipsis menu.